src/utils.js

```diff
@@ -22,3 +22,11 @@ export function preserveRef(ref, node) {
 export function ssrSafeCreateDiv() {
   return typeof document !== 'undefined' && document.createElement('div')
 }
+
+export function updateClassNames(classNames, action, tooltip) {
```

Comment:
I think `updateClassName` (without `s`) is fine. Also parameter ordering probably should be: `tooltip, action, classNames`. 

Also I think it should be `.split(/\s+/)` to catch multiple whitespaces?

Reply:
Great stuff, thanks for the feedback! Agreed on all points. I'll try to patch it up at lunch today.